backend/src/email/utils/query.ts

```diff
@@ -1,13 +0,0 @@
-import { EmailBlacklist } from '@email/models'
```

Comment:
deleted this and included in `EmailService`
- fits better there
- allows me to mock the return value